core/utils/silly-i18n/src/index.js

```diff
@@ -90,10 +43,19 @@ export class I18n {
                     " is not a key in the translations hash provided"
             );
         }
+
         this.language = language in translations ? language : fallbackLanguage;
         this._translations = translations[this.language];
     }
 
+    /**
+     * Know which sections are present in the original hash
+     * @returns The translated string.
+     */
+    sections() {
```

Comment:
Considering how `this._translations` shouldn't be modified after construction, couldn't this simply be a property, rather than a function?